pom.xml

```diff
@@ -182,14 +181,12 @@
 
                             <tags>
                                 <tag>jdbc</tag>
-                                <tag>java</tag>
                                 <tag>database</tag>
                                 <tag>schema evolution</tag>
```

Comment:
I'd also remove `schema-evolution`. Sorry for missing that originally.

Reply:
Also, I wonder if we should include tags for at least some of the more popular DBMSes: `mysql`, `postgresql`, `oracle`, `sql server`, `db2`, `sybase`, `vertica`, `sap hana`, `derby`, `sqlite`.